ReactiveCocoa/Swift/Flatten.swift

```diff
@@ -92,6 +198,53 @@ extension SignalType where Value: SignalType, Error == Value.Error {
 	}
 }
 
+extension SignalType where Value: SignalType, Error == NoError {
+	/// Flattens the inner signals sent upon `signal` (into a single signal of
+	/// values), according to the semantics of the given strategy.
+	///
+	/// If an active inner signal emits an error, the returned signal will
+	/// forward that error immediately.
+	///
+	/// `Interrupted` events on inner signals will be treated like `Completed`
+	/// events on inner signals.
+	@warn_unused_result(message="Did you forget to call `observe` on the signal?")
+	public func flatten(strategy: FlattenStrategy) -> Signal<Value.Value, Value.Error> {
+		return self
+			.map(SignalProducer.init)
```

Comment:
We can omit this line and use [the impl](https://github.com/ReactiveCocoa/ReactiveCocoa/pull/2885/files#diff-b06d9f2cdc22697e0202bb4fa92ca00dR196).